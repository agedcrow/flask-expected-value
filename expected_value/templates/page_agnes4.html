{% extends "base.html" %}
{% block content %}

  <div class="p-4 text-center">
    <h2 class="fs-5 mt-2 mb-4 text-secondary">{{ title }}</h2>

    <p>Spec</p>
    <div class="mb-4 mx-auto" style="width:fit-content;">{{ tbl_spec | safe }}</div>

    {% for start_game, tbl_border in tbl_borders %}
      <p>{{ start_game }}</p>
      <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border | safe }}</div>
    {% endfor %}

    {# <p>{{ start_game0 }}</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border0 | safe }}</div>
    
    <p>{{ start_game1 }}</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border1 | safe }}</div>

    <p>{{ start_game2 }}</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border2 | safe }}</div>
    
    <p>{{ start_game3 }}</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border3 | safe }}</div>

    <p>{{ start_game4 }}</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border4 | safe }}</div>
    
    <p>{{ start_game5 }}</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border5 | safe }}</div>

    <p>{{ start_game6 }}</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border6 | safe }}</div>
    
    <p>{{ start_game7 }}</p>
    <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border7 | safe }}</div> #}

    {# <p>Result</p>
    <div class="mb-2 mx-auto" style="width:fit-content;">{{ tbl_result | safe }}</div>
    
    <div class="mb-4">
      <img src="data:image/png;base64,{{ img_data }}" alt="Image", width="420"/>
    </div> #}

  </div>

{% endblock%}