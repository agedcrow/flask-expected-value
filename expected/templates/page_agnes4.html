{% extends "base.html" %}
{% block content %}

  <div class="p-4 text-center">
    <h2 class="fs-5 mt-2 mb-4">{{ title }}</h2>

    <p>Specs</p>
    <div class="mb-4 mx-auto" style="width:fit-content;">{{ tbl_specs | safe }}</div>

    <p>Rate tables</p>
    {% for caption, tbl_border in tbl_borders %}
      <div class="mb-1">{{ caption }}</div>
      <div class="mb-4 mx-auto" style="width:fit-content">{{ tbl_border | safe }}</div>
    {% endfor %}

    <p>Result</p>
    <div class="mb-2 mx-auto" style="width:fit-content;">{{ tbl_result | safe }}</div>
    
    <div class="mb-4">
      <img src="data:image/png;base64,{{ plot_img }}" alt="Image", width="420"/>
    </div>

  </div>

{% endblock%}